widgetssdk/src/main/java/com/glia/widgets/config/components/Layer.java

```diff
@@ -0,0 +1,43 @@
+package com.glia.widgets.config.components;
+
+import androidx.annotation.Nullable;
+
+import com.glia.widgets.config.components.base.DpSize;
+import com.google.gson.annotations.SerializedName;
+
+public class Layer {
+    @SerializedName("color")
+    @Nullable
+    private RemoteColor color;
+    @SerializedName("border")
+    @Nullable
+    private RemoteColor border;
+
+    @SerializedName("borderWidth")
+    @Nullable
+    private DpSize borderWidth;
+
+    @SerializedName("cornerRadius")
+    @Nullable
+    private DpSize cornerRadius;
+
+    @Nullable
+    public RemoteColor getColor() {
+        return color;
+    }
+
+    @Nullable
+    public RemoteColor getBorder() {
```

Comment:
NIT: I think `getBorderColor()` would be safer for scalability and more consistent with other border properties